analysis-engine/src/test/java/org/sonarsource/sonarlint/core/analysis/mediumtests/AnalysisEngineMediumTests.java

```diff
@@ -201,6 +207,18 @@ void should_interrupt_executing_thread_when_stopping() {
     assertThat(futureLongCommand).isCompletedWithValue("INTERRUPTED");
   }
 
+  @Test
+  void should_not_log_any_error_when_stopping() {
```

Comment:
Is the test failing if you remove the fix?

Reply:
I checked of course :wink: